.github/workflows/cyclonus-netpol-test.yaml

```diff
@@ -0,0 +1,53 @@
+name: Cyclonus Network Policy Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    # run once a day at midnight
+    - cron: '0 0 * * *'
+
+jobs:
+  cyclonus-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Make NPM image
+        run: |
+          VERSION=cyclonus make azure-npm-image
+
+      - name: Setup Kind
+        uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.9.0"
+          config: ./test/kind/kind.yaml
+          name: npm-kind
+      
+      - name: Install Azure NPM
+        run: |
+          sed -i 's/mcr.microsoft.com\/containernetworking\/azure-npm:v1.3.1/acnpublic.azurecr.io\/azure-npm:cyclonus/' ./npm/azure-npm.yaml
+          kind load docker-image acnpublic.azurecr.io/azure-npm:cyclonus --name npm-kind
```

Comment:
Do you think using the same tag will cause issues when we are trying to run multiple cyclonus tests together ? (if not today ofr later)

Reply:
Since the cluster used is kind based on the same host where the images are built, and the images are directly loaded on to the kind nodes, there won't be conflict with parallel tests